kaitai_utils.py

```diff
@@ -8,6 +8,8 @@
 import hansken_extraction_plugin
 from hansken_extraction_plugin.api.extraction_trace import ExtractionTrace
 import json
+
+from hansken_extraction_plugin.api.transformation import RangedTransformation, Range
```

Comment:
I think the import order is a little weird here, you can use ctrl + alt + O to organize imports.

Reply:
Cool, done!